notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1593_force_actuator_error_elevation.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    n_bins = 30
</pre><p>See that <code>n_bins</code> is repeated here. I would delete this one once the other is setup earlier in the notebook.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
OK!